libs/ui/src/tokens/components/atoms/_icon.css

```
@@ -194,3 +198,13 @@
 @utility token-icon-search {
   @apply icon-[mdi--magnify];
 }
+
+/* Semantic action aliases decoupled from concrete icon glyphs. */
+
+@utility token-icon-increment {
```

Stylelint (17.9.0)

[error] Unexpected unknown at-rule "@utility" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)